src/tests/AttributesManagersTest.cpp

```diff
@@ -568,7 +568,7 @@ void AttributesManagersTest::testPhysicsAttributesManagersCreate() {
 
 void AttributesManagersTest::testStageAttributesManagersCreate() {
   std::string stageConfigFile =
-      Cr::Utility::Path::join(DATA_DIR, "test_assets/scenes/simple_room.glb");
+      Cr::Utility::Path::join(DATA_DIR, "test_assets/scenes/plane.glb");
```

Comment:
@jturner65 seems like this is testing construction of a stage Attributes from a raw .glb with no config, so this change should be fine.

Reply:
Yes, the test needs to have a stage attributes that corresponds to an existing asset, so it is creating the default.